### ISSUE TITLE
'EditorUserBuildSettings' does not contain a definition for 'wsaGenerateReferenceProjects'

### ISSUE THREAD

**Issue by einstyle** [Microsoft/MixedRealityToolkit#219]
Hi,

I tried to import HoloToolkit 2017.4 into a new unity project (Unity 2019.1), and I have this error: error CS0117: 'EditorUserBuildSettings' does not contain a definition for 'wsaGenerateReferenceProjects'

Any suggestions to solve this?

**Comment by chrisfromwork** [comment]
assigning to @Yoyozilla. I believe moving forward that we may not be supporting the HoloToolkit.

**Comment by StephenHodgson** [comment]
Yup, agreed for HoloToolkit I think the highest support should be for 2018
